docs/digger/installation/install-android-sdk.asciidoc

```diff
@@ -1,21 +1,23 @@
+//pmacko: +1, changed to active form
+WARNING: During the process, you will be ask to manually accept the Android SDK License Agreement.
+ 
+The Android SDK is required to build Android apps. The installer downloads the Android SDK and makes it available to the building pods by performing the following steps:
 
-To build Android applications, Jenkins will dynamically create containers (pods) on OpenShift using the https://wiki.jenkins.io/display/JENKINS/Kubernetes+Plugin[Kubernetes Jenkins plugin]. This approach has a few advantages:
+* Creating a pod which will be used to download the Android SDK. The pod will use an image that contains the required tools, and claim a Persistent Volume (PV).
+* Invoke a script to download the Android SDK. The Android SDK will be retained in the PV.
+* Update the Kubernetes plugin configuration to add a new pod template. It will mount and connect the PV in “Read-Only” mode to the pods, and setup the required environment variables to define an Android home directory.
 
-* It doesn't require a build server to be running constantly
-* Each build is performed completely in isolation
+Jenkins dynamically creates containers (pods) on OpenShift using the https://wiki.jenkins.io/display/JENKINS/Kubernetes+Plugin[Kubernetes Jenkins plugin] for each Android build.
 
-The Android SDK is required to build Android apps. The installer will download the Android SDK and make it available to the building pods by performing the following tasks:
+//* It doesn't require a build server to be running constantly
+//* Each build is performed completely in isolation -> redundant I think, good for "marketing", otherwise I think not, ppl are familiar with containers
 
-* Creating a new pod which will be used to download the Android SDK. The pod will use an image that contains the required tools, and claim a Persistent Volume(PV).
-* Invoke a script to download the Android SDK. During the process, the user will be prompted to accept the Android SDK License Agreement. The Android SDK will be persisted in the PV.
-* Update the Kubernetes plugin configuration to add a new pod template. It will mount the PV in “Read-Only” mode to the pods and setup the required environment variables to define an Android home directory.
 
-==== Configure the Android SDK components 
+==== Configure Android SDK components 
 
-The Android SDK contains many different components. 
-Only a subset of them are installed by default.
-The list of components that will be installed are specified https://github.com/aerogear/digger-installer/blob/master/android-sdk/templates/sample_cfg.j2[in this file] contained in digger-installer repo .
+The Android SDK contains many different components. Only a subset of them are installed by default.
+The list of installed components are specified https://github.com/aerogear/digger-installer/blob/master/android-sdk/templates/sample_cfg.j2[in this file] contained in `digger-installer` repo .
```

Comment:
Dont think pointing them at the jinja file is a good idea, best to refer to vars, we do not want them editing this file

Reply:
yes agreed, this was commented on in the google doc. will resolve